EKF/estimator_base.cpp

```diff
@@ -259,27 +258,6 @@ void EstimatorBase::initialiseVariables(uint64_t time_usec)
 	_state.quat_nominal.setZero();
 	_state.quat_nominal(0) = 1.0f;
 
-	_params.mag_delay_ms = 0;
```

Comment:
Why are you now not initialising the values to sane values? The filter as a software component is unusable without those sane defaults.